<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOriginal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelChanged">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="chuong2Group">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Chương 2</string>
    </property>
    <widget class="QDial" name="rotation_inp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rotation_val">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0°</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Scaling</string>
     </property>
    </widget>
    <widget class="QSlider" name="scaling_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="scaling_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tranX_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transtation</string>
     </property>
    </widget>
    <widget class="QSlider" name="tranX_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="tranY_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="tranY_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Affine</string>
     </property>
    </widget>
    <widget class="QSlider" name="affine_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="affine_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="chuong4Group">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>330</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Chương 4</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Median</string>
     </property>
    </widget>
    <widget class="QSlider" name="median_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>13</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="median_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gaussian</string>
     </property>
    </widget>
    <widget class="QSlider" name="gaussian_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>13</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="gaussian_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="highboost_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QSlider" name="highboost_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>8</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
     <property name="sliderPosition">
      <number>8</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>High Boost</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="average_val">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Average</string>
     </property>
    </widget>
    <widget class="QSlider" name="average_inp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>13</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>80</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Chương 3</string>
    </property>
    <widget class="QLineEdit" name="log_val">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QSlider" name="log_inp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Changed Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1365</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionChuong2"/>
    <addaction name="actionChuong3"/>
    <addaction name="actionChuong4"/>
    <addaction name="actionChuong5"/>
    <addaction name="actionChuong7"/>
    <addaction name="actionChuong8"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/Image_Upload.png</normaloff>images/Image_Upload.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionChuong1">
   <property name="text">
    <string>Chương 1</string>
   </property>
  </action>
  <action name="actionChuong2">
   <property name="text">
    <string>Chương 2</string>
   </property>
  </action>
  <action name="actionChuong3">
   <property name="text">
    <string>Chương 3</string>
   </property>
  </action>
  <action name="actionChuong4">
   <property name="text">
    <string>Chương 4</string>
   </property>
  </action>
  <action name="actionChuong5">
   <property name="text">
    <string>Chương 5</string>
   </property>
  </action>
  <action name="actionChuong7">
   <property name="text">
    <string>Chương 7</string>
   </property>
  </action>
  <action name="actionChuong8">
   <property name="text">
    <string>Chương 8</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
